paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/networking/Models.kt

```diff
@@ -34,22 +33,21 @@ fun CartState.toCheckoutRequest(): ExampleCheckoutRequest {
 }
 
 @Serializable
-@Keep
 data class ExampleCheckoutResponse(
-    @SerializedName("publishableKey")
+    @SerialName("publishableKey")
     val publishableKey: String,
-    @SerializedName("paymentIntent")
-    val paymentIntent: String,
-    @SerializedName("customer")
+    @SerialName("paymentIntent")
+    val paymentIntent: String = "",
```

Comment:
Why do some need a default value and some don't?

Reply:
They aren't being sent by the server (I think), gson was doing implicit defaults, kotlinx.serialization requires explicit defaults.

Reply:
Should we have explicit defaults for all of them then?

Reply:
Just the ones that the server won't send. Failing is a benefit if the server doesn't send something we expect (in my opinion).